src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -889,13 +889,13 @@ private void BeginExecuteNonQueryInternalReadStage(TaskCompletionSource<object>
             }
         }
 
-        private void VerifyEndExecuteState(Task completionTask, String endMethod)
+        private void VerifyEndExecuteState(Task asyncResult, String endMethod)
```

Comment:
Since the public `Begin`/`End` methods don't exist in .NET Core, the `Task` passed in here should never be null, so I've replaced the `ArgumentNullException` with a `Debug.Assert` to avoid the parameter rename altogether.